app/controllers/homepages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomepagesController < ApplicationController
+  def index
+    def index
```

Comment:
Hm?! Why is an `index` method defined inside of the `index` method?! Actually, this is what produced the really interesting bug!!